app/views/govuk_publishing_components/components/feedback/_yes_no_banner.html.erb

```diff
@@ -6,62 +6,29 @@
   <div class="gem-c-feedback__prompt-questions js-prompt-questions">
     <h2 class="gem-c-feedback__prompt-question"><%= t("components.feedback.is_this_page_useful", default: "Is this page useful?") %></h2>
     <!-- Maybe button exists only to try and capture clicks by bots -->
-    <%= link_to t("components.feedback.maybe", default: "Maybe"), contact_govuk_path, {
-        class: 'gem-c-feedback__prompt-link',
-        data: {
-          'track-category' => 'yesNoFeedbackForm',
-          'track-action' => 'ffMaybeClick'
-        },
-        'aria-expanded': false,
-        role: 'button',
-        style: 'display: none;',
-        hidden: 'hidden',
-        'aria-hidden': 'true',
-      } %>
+    <button class="govuk-button govuk-button--secondary gem-c-feedback__prompt-link" data-track-category="yesNoFeedbackForm" data-track-action="ffMaybeClick" aria-expanded="false" style="display: none" hidden="hidden" aria-hidden="true">
+      <%= t("components.feedback.maybe", default: "Maybe") %>
+    </button>
+    
     <ul class="gem-c-feedback__option-list">
       <li class="gem-c-feedback__option-list-item">
-        <%= link_to contact_govuk_path, {
-          class: 'gem-c-feedback__prompt-link js-page-is-useful',
-          data: {
-            'track-category' => 'yesNoFeedbackForm',
-            'track-action' => 'ffYesClick'
-          },
-          role: 'button',
-        } do %>
+        <button class="govuk-button govuk-button--secondary gem-c-feedback__prompt-link js-page-is-useful" data-track-category="yesNoFeedbackForm" data-track-action="ffYesClick">
           <%= t("components.feedback.yes", default: "Yes") %> <span class="govuk-visually-hidden"><%= t("components.feedback.is_useful", default: "this page is useful") %></span>
-        <% end %>
+        </button>
       </li>
       <li class="gem-c-feedback__option-list-item">
-        <%= link_to contact_govuk_path, {
-          class: 'gem-c-feedback__prompt-link js-toggle-form js-page-is-not-useful',
-          data: {
-            'track-category' => 'yesNoFeedbackForm',
-            'track-action' => 'ffNoClick'
-          },
-          'aria-controls': 'page-is-not-useful',
-          'aria-expanded': false,
-          role: 'button',
-        } do %>
+        <button class="govuk-button govuk-button--secondary gem-c-feedback__prompt-link js-toggle-form js-page-is-not-useful" data-track-category="yesNoFeedbackForm" data-track-action="ffNoClick" aria-controls="page-is-not-useful" aria-expanded="false">
```

Comment:
This all looks sound. I think my only problem is that due to the changes the Yes/No buttons collapse onto a new line much earlier than they used to (see below). This isn't the end of the world, so I'm not certain whether it should hold this up or not.

<img width="912" alt="Screenshot 2020-09-22 at 17 21 53" src="https://user-images.githubusercontent.com/861310/93909928-45b80c00-fcf8-11ea-8707-08d2b6d026cf.png">

I've roughly prototyped a possible fix, using `display: table` on the subwrapper, and `display: table-cell` and `vertical-align: bottom` on the two wrapping half-divs. Would look like this:

<img width="921" alt="Screenshot 2020-09-22 at 17 24 58" src="https://user-images.githubusercontent.com/861310/93910218-b0694780-fcf8-11ea-8920-b879172c9fea.png">

Happy to discuss.

Reply:
It's a bit late now and you ain't wrong, so I'd rather it be good than not. I'll have a play around tonight and see if I can make this a little tidier. I've got my sights on that big ugly space above `Is this page useful` as well.

Reply:
Great, and happy to help if you fancy pairing on it when things calm down a little. In the meantime, I approve this message.

Reply:
Fixed the design for this awkward area between tablet and desktop. Let me know what you think, happy to chat for real.

Reply:
Niiiiice. It's slightly wonky on IE10/11, but I think forgivable. Certainly much better than it was 👍 

<img width="1096" alt="Screenshot 2020-09-23 at 12 01 58" src="https://user-images.githubusercontent.com/861310/94004462-a72db880-fd94-11ea-975a-353f121dd728.png">